src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => {
+  <div className="catagories">
+    <button className="status_button" type="button">Check Status</button>
+  </div>;
+};
```

Comment:
- [ ] To render this button, the return value should be open with `()` and not `{}`. For example
```
const Categories = () => (
  <div className="catagories">
    <button className="status_button" type="button">Check Status</button>
  </div>
);
```